packages/frontend/src/fixtures/stubs.ts

```diff
@@ -0,0 +1,132 @@
+export const trustRegistryStub = {
+  entities: [
+    {
+      name: 'Absa',
+      dids: ['did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb'],
+      logo_url:
+        'https://s3.eu-central-1.amazonaws.com/builds.eth.company/absa.svg',
+      domain: 'www.absa.africa',
+      role: ['issuer'],
+      credentials: [
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      ],
+      id: '8fa665b6-7fc5-4b0b-baee-6221b1844ec8',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+    {
+      name: 'Example entity',
+      dids: ['did:indy:sovrin:staging:Pmi5CXJbv4HM8YttyomMoW'],
+      logo_url:
+        'https://s3.eu-central-1.amazonaws.com/builds.eth.company/absa.svg',
+      domain: 'www.example.com',
+      role: ['issuer'],
+      credentials: [
+        'did:indy:sovrin:staging:Pmi5CXJbv4HM8YttyomMoW/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      ],
+      id: '6na665b6-7fc5-4b0b-baee-6221b1844ea7',
+      createdAt: '2023-11-10T14:43:36.083Z',
+      updatedAt: '2023-11-20T16:49:58.307Z',
+    },
+  ],
+  schemas: [
+    {
+      schemaId:
+        'did:indy:sovrin:staging:C279iyCR8wtKiPC8o9iPmb/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      name: 'Digital Identity',
+      createdAt: '2023-11-10T14:41:49.143Z',
+      updatedAt: '2023-11-20T16:49:09.838Z',
+    },
+    {
+      schemaId:
+        'did:indy:sovrin:staging:Pmi5CXJbv4HM8YttyomMoW/anoncreds/v0/SCHEMA/e-KYC/1.0.0',
+      name: 'Digital Identity 2',
```

Comment:
We should use real data as much as possible. What about Phone Number or Bank Account credential?